package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "instant",
+  "version": "0.0.12",
+  "main": "instant.ts",
+  "repository": "git@github.com:openhie/instant.git",
+  "author": "Jembi and IntraHealth",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@types/command-line-args": "^5.0.0",
+    "@types/glob": "^7.1.3",
+    "@types/node": "^14.0.22",
+    "command-line-args": "^5.1.1",
+    "env-cmd": "^10.1.0",
+    "glob": "^7.1.6",
+    "ts-node": "^8.10.2",
+    "typescript": "^3.9.6"
+  },
+  "scripts": {
+    "instant": "ts-node instant.ts",
+    "test:local": "docker run --rm --name test-helper -v instant:/instant --network instant_default openhie/package-test local",
+    "test:local:dev": "docker run --rm --name test-helper -v </absolute/path/to/instant>:/instant --network instant_default openhie/package-test local",
+    "test:remote": "docker run --rm --name test-helper --network host -v instant:/instant -v </absolute/path/to/instant/.env.remote>:/instant/.env.remote openhie/package-test remote",
+    "test:remote:dev": "docker run --rm --name test-helper --network host -v </absolute/path/to/instant>:/instant openhie/package-test remote",
+    "test:container": "./test.sh",
+    "docker:build": "docker build -t openhie/instant:latest .",
```

Comment:
Seems like all this with specific reference to one of the repos that use the cli should be removed from here. No more references to instant...

Reply:
: